requirements/test-requirements.txt

```diff
@@ -43,4 +43,5 @@ databricks-sdk
 # TODO: Unpin once we support openai 1.0
 openai<1.0
 # Required for showing pytest stats
-psutil
\ No newline at end of file
+psutil
+typing_extensions>=4.6.0
```

Comment:
Q: shall we update core-requirement.txt file ?

Reply:
discussion:

core-requirement.txt has this line `sqlalchemy<3,>=1.4.0`,

typing_extensions>=4.6.0 should be set as dependency requirement of `sqlalchemy` package.

shall we ask `sqlalchemy` team to address this first, and for now we exclude `2.0.25` version in MLflow core-requirements for now ?

Reply:
It's updated in their master branch and will be released in 2.1 in the next few weeks. Of note on that release as well: they're dropping python 3.8 support in 2.1. 
Based on this, I think this PR is fine for now as the dependency update is only going to be applied to the next new version which we won't be able to use until we drop support for Python 3.8